teste/SME.SGP.TesteIntegracao/Aula/ServicosFakes/ServicoAuditoriaFakeAdministrador.cs

```diff
@@ -0,0 +1,21 @@
+using System.Threading.Tasks;
+using SME.SGP.Dominio;
+using SME.SGP.Dominio.Constantes.MensagensNegocio;
+using SME.SGP.Infra.Interface;
+
+namespace SME.SGP.TesteIntegracao.Aula.ServicosFakes
+{
+    public class ServicoAuditoriaFakeAdministrador: IServicoAuditoria
+    {
+        public ServicoAuditoriaFakeAdministrador()
+        {}
+
+        public async Task<bool> Auditar(Auditoria auditoria)
+        {
+            if (!auditoria.Administrador.Equals("7924488"))
+                throw new NegocioException(MensagemNegocioComuns.A_AUDITORIA_NAO_FOI_REGISTRADA_PELO_ADMINISTRADOR);
```

Comment:
o que significa administrador diferente de 7924488 ? não ta claro o que faz isso.
acredito que a regra não seja dar exceção se for auditar com um usuário específico

Reply:
Resolvido